app/client/src/pages/Editor/GlobalHotKeys.tsx

```diff
@@ -75,13 +72,9 @@ class GlobalHotKeys extends React.Component<Props> {
           global
           label="Search entities"
           onKeyDown={(e: any) => {
-            const entitySearchInput = document.getElementById(
-              ENTITY_EXPLORER_SEARCH_ID,
-            );
             const widgetSearchInput = document.getElementById(
               WIDGETS_SEARCH_ID,
             );
-            if (entitySearchInput) entitySearchInput.focus();
             if (widgetSearchInput) widgetSearchInput.focus();
             e.preventDefault();
             e.stopPropagation();
```

Comment:
Minor change here: so `cmd+f` works when the widget search does not exist

```suggestion
            if (widgetSearchInput) {
              widgetSearchInput.focus();
              e.preventDefault();
              e.stopPropagation();
            }
```